content/hi/docs/reference/glossary/service.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Service
```

Comment:
This should be in Hindi

Reply:
You can use `aka:` to also provide the original name.